docs/hoon/twig/ket-cast/lus-like.md

```diff
@@ -16,11 +16,13 @@ within the span of `q`.  Otherwise, `nest-fail`.
 
 Regular: *2-fixed*.
 
-Irregular: `\`_foo\`bar` is `:like(foo bar)`.
-
 ## Examples
 
 ```
-~zod:dojo> :like('text' 97)
+~zod:dojo> :like('text' %a)
 'a'
+~zod:dojo> :like('text' 97)
+nest-fail
 ```
```

Comment:
If it's broken we should probably take it out — but I'm not even sure what that was initially meant to do.